### PR TITLE
Fixed #98 - failing post tests 

### LINKED ISSUE #98: One of the tests written for Posts fails for no reason
## About

* **Related Issues:** #91 

When I run the tests the test named `test_existing_post` fails. But I can see that posts return fine when I run the API itself.

### PR DESCRIPTION
**Pull request by iremUlutas**
Update data_template in order to have a default fields of type JSONField.

## About

* **Related Issues:** 98

Bug Fix with Failing Post Tests.

### Changes
Update data_template model to have a default value.

### CODE REVIEW

File: api/interestr/api/tests.py [hunk 1]
```diff
@@ -43,11 +48,13 @@ def test_existing_post(self):
 
 		json_response = json.loads(response.content)
 
-		self.assertEqual(json_response, {'owner' : self.test_user.id, 'text' : 'text', 'content_url' : '/post/1', 'group' : None})
+		#self.assertEqual(json_response['owner'], self.test_user.id)
```
Comment (karacasoft) [inline api/interestr/api/tests.py]: Remove commented out codes

File: api/interestr/api/tests.py [hunk 2]
```diff
@@ -34,7 +35,11 @@ class PostTests(TestCase):
 	def setUp(self):
 		self.client = Client()
 		self.test_user = User.objects.create_user('owner', 'wow@wow.com', 'wowpass123')
-		self.test_post = Post.objects.create(owner = self.test_user, text = 'text', content_url = '/post/1')
+		self.test_group = Group.objects.create(name = "test_group", description="lalala")
+		self.test_data_template = DataTemplate.objects.create(name="",group=self.test_group, user=self.test_user)
+		#self.test_post = Post.objects.create(owner = self.test_user, text = 'text', data_template = self.test_data_template, data = self.JSONField)
```
Comment (karacasoft) [inline api/interestr/api/tests.py]: Remove commented out codes 2

File: api/interestr/data_templates/models.py [hunk 3]
```diff
@@ -22,7 +22,7 @@ class DataTemplate(core_models.BaseModel):
     group = models.ForeignKey(core_models.Group, on_delete=models.SET_NULL, null=True)
     user = models.ForeignKey(auth_models.User, on_delete=models.SET_NULL, null=True)
 
-    fields = JSONField()
+    fields = JSONField(default=None, null=True)
```
Comment (karacasoft) [inline api/interestr/data_templates/models.py]: I think we should not allow DataTemplates with no fields. What do you think @smddzcy ?

Comment (smddzcy) [reply api/interestr/data_templates/models.py]: @triforce930 Definitely.

**Comment by smddzcy** [comment]
I just changed many things about data templates, see #123. We'll probably need to change some things in this PR 😄

**Comment by karacasoft** [comment]
This must be from a very old branch that we never merged into master. lol. Can you open up a new branch from master, make your changes and open another request?

**Comment by karacasoft** [comment]
I'm thinking, no one will ever review this mess. I suggest you to open up a new branch from master and carry over your changes for the tests there. I think you can do this with git-rebase, too, but it'll probably just increase the probability of things getting worse.

**Pull request closed without merging**